draft-amend-tsvwg-multipath-dccp.mkd

```diff
@@ -283,20 +326,20 @@ congestion control.
 | Session handover             | yes                   | yes                |
 | Path aggregation             | yes                   | yes                |
 | Robust session establishment | no                    | yes                |
-| Data reassembly              | yes                   | optional / modular |
+| Data re-assembly             | yes                   | optional / modular |
```

Comment:
Indifferent on this - some texts use "reassembly" whilst others use "re-assembly". Important thing is being self consistent

Reply:
Yep, and I think in our document it is consistent with "re-assembly". At least I checked again the whole document for some legacy "reassembly"